src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck starts working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped working");

```

Comment:
Similarly, the message 'Truck stopped working' could provide more information about what it means for the Truck to stop working.